apps/impala/src/impala/api.py

```diff
@@ -142,20 +144,27 @@ def alanize(request):
     doc = Document2.objects.get(id=query_id)
     snippets = doc.data_dict.get('snippets', [])
     secret = snippets[0]['result']['handle']['secret']
-    impala_query_id = unpack_guid(base64.decodestring(secret))
+    impala_query_id = unpack_guid(decodebytes(secret))
```

Comment:
@Harshg999 had made a similar switch to using `base64.b64decode` in #2949 . How does `base64.decodebytes` relate to `base64.b64decode`?

Reply:
Good catch, there may be similarities between them.
Would suggest to take a look at both and compare what's best to use (might help in any last minute surprises).

Reply:
decodebytes -> Decode the bytes-like object , which must contain one or more lines of base64 encoded data, and return the decoded bytes
b64decode -> Decode the Base64 encoded bytes-like object or ASCII string s and return the decoded bytes.

--> b64decode can take inputs string and byte both but decodebytes only takes bytes. Looks like it was wrong earlier also need to use b64decode.